### PR TITLE
Fix the channel-action when the text is too long

### LINKED ISSUE #19438: The "Pinned messages" button displaces the "Mute chat" button if a long message is pinned
# Bug Report

## Problem 

Screenshot 2024-03-28 at 14 46 50

Screenshot 2024-03-28 at 14 48 38

#### Expected behavior

The button has a fixed width regardless of the content

### Reproduction

1. Pin a long message in any chat/channel
2. Check the top of the chat

### Additional Information

- Status version: RC 2.28 
- Operating System: Android, iOS

[comment]: # (Please, add logs/notes if necessary)

### PR DESCRIPTION
**Pull request by clauxx**
fixes #19438

### Summary
Fix the size of the large channel action if the text is too long

### Steps to test

check out the channel actions preview or ...

- Open Status
- open chat
- pin large message
- look at channel actions

status: ready

### CODE REVIEW

File: src/quo/components/community/channel_action/style.cljs [hunk 1]
```diff
@@ -6,7 +6,7 @@
   [{:keys [big? disabled?]}]
   (cond-> {}
     disabled?  (assoc :opacity 0.3)
-    big?       (assoc :flex-grow 1)
+    big?       (assoc :flex-grow 1 :flex-shrink 1)
```
Comment (shivekkhurana) [inline src/quo/components/community/channel_action/style.cljs]: This is essentially `:flex 1`

File: src/status_im/contexts/preview/quo/community/channel_actions.cljs [hunk 2]
```diff
@@ -34,6 +34,17 @@
...
        :icon                :i/muted}]}]
    [rn/view {:height 50}]
+   [quo/channel-actions
+    {:actions
+     [{:big? true
+       :label "Pinned Messages sometimes can get very long and we have to take care of that"
+       :customization-color :blue
+       :icon :i/pin
+       :counter-value 5}
+      {:label               "Mute chat"
```
Comment (shivekkhurana) [inline src/status_im/contexts/preview/quo/community/channel_actions.cljs:44]: Should not be string, but a reference to translation.

File: src/status_im/contexts/preview/quo/community/channel_actions.cljs [hunk 3]
```diff
@@ -34,6 +34,17 @@
        :customization-color :blue
        :icon                :i/muted}]}]
    [rn/view {:height 50}]
+   [quo/channel-actions
+    {:actions
+     [{:big? true
+       :label "Pinned Messages sometimes can get very long and we have to take care of that"
```
Comment (shivekkhurana) [inline src/status_im/contexts/preview/quo/community/channel_actions.cljs:40]: Should be translation.

File: src/status_im/contexts/preview/quo/community/channel_actions.cljs [hunk 4]
```diff
@@ -34,6 +34,17 @@
        :customization-color :blue
        :icon                :i/muted}]}]
    [rn/view {:height 50}]
+   [quo/channel-actions
+    {:actions
```
Comment (shivekkhurana) [inline src/status_im/contexts/preview/quo/community/channel_actions.cljs:38]: Is it possible to add accessibility labels for QA team?

**Review by shivekkhurana — CHANGES_REQUESTED** [review]
.

File: src/status_im/contexts/preview/quo/community/channel_actions.cljs (hunk 2 above)
Comment (clauxx) [reply src/status_im/contexts/preview/quo/community/channel_actions.cljs:44]: this is a preview screen

File: src/status_im/contexts/preview/quo/community/channel_actions.cljs (hunk 3 above)
Comment (clauxx) [reply src/status_im/contexts/preview/quo/community/channel_actions.cljs:40]: this is a preview screen

File: src/quo/components/community/channel_action/style.cljs (hunk 1 above)
Comment (clauxx) [reply src/quo/components/community/channel_action/style.cljs]: done

File: src/status_im/contexts/preview/quo/community/channel_actions.cljs (hunk 4 above)
Comment (clauxx) [reply src/status_im/contexts/preview/quo/community/channel_actions.cljs:38]: done

**Comment by clauxx** [comment]
> .

@shivekkhurana please have a look again

**Review by Parveshdhull — APPROVED** [review]

**Review by shivekkhurana — APPROVED** [review]

**Comment by pavloburykh** [comment]
@clauxx thank you fro the fix. Ready for merge.

**Review by BalogunofAfrica — APPROVED** [review]

**Pull request merged**